Target.java

```diff
@@ -1,17 +1,63 @@
-/*
- * File: Target.java
- * Name: 
- * Section Leader: 
- * -----------------
- * This file is the starter file for the Target problem.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Target extends GraphicsProgram {	
-	public void run() {
-		/* You fill this in. */
-	}
-}
+/*
+ * File: Target.java
+ * Name: 
+ * Section Leader: 
+ * -----------------
+ * This file is the starter file for the Target problem.
+ */
+
+import acm.graphics.*;
+import acm.program.*;
+import java.awt.*;
+
+public class Target extends GraphicsProgram {
+	
+ 
+	public void run(double OUTTERRad) {

```

Comment:
I don't think you should be passing an argument to the run method. Also you should follow naming convention. You can't have ALL caps mixed with non small letters. So you can't have this:
OUTTERRad or outterRAD...
Your argument should be: outterRad (This naming convention is called Camel case naming convention).